inventory/inventoryService.js

```diff
@@ -0,0 +1,16 @@
+import { FailedGettingError } from '../errors/errorTypes/failedGettingError.js';
+import { FAILED_GETTING, INVENTORY } from '../utils/textConstants.js';
+
+export class InventoryService {
+  constructor ({ inventoryRepository }) {
+    this.inventoryRepository = inventoryRepository;
+  }
+
+  getAll = async (id) => {
```

Comment:
Be more specific in the name like: getAllUserItems(userId) or getAllItemsPerUser